build/artifacts.props

```diff
@@ -147,7 +147,7 @@
     <PackageArtifact Include="Microsoft.AspNetCore.Routing.Abstractions" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
     <PackageArtifact Include="Microsoft.AspNetCore.Routing.DecisionTree.Sources" Category="noship" />
     <PackageArtifact Include="Microsoft.AspNetCore.Server.HttpSys" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
-    <PackageArtifact Include="Microsoft.AspNetCore.Server.IIS" Category="ship" WinAppMetapackage="true" WinAllMetapackage="true" />
+    <PackageArtifact Include="Microsoft.AspNetCore.Server.IIS" Category="noship" WinAppMetapackage="true" WinAllMetapackage="true" />
```

Comment:
We'll need to cut out more than just this to unship it. You'll need to revert https://github.com/aspnet/Universe/pull/989 too.

Reply:
Hey, let's think about the positives. There will be no native dependencies in the meta packages now!